build/linux/installer/scripts/livenessprobe.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
should this just be a rename?

Reply:
> should this just be a rename?

just did diff with whats there is in ci_feature so no differences so we are good on this.